app/Bots/Commands/QuoteCommand.php

```diff
@@ -163,4 +168,9 @@ public function handle()
             'disable_notification' => true,
         ]);
     }
+
+    private function quoteDuplicate($quote)
+    {
+        return BotQuote::where('quote', $quote)->exists();
```

Comment:
While this definitely works, Gumbo users sometimes like to submit quotes in duplicate.

Maybe adding a `message_id` column to the `bot_quotes` table and using the `getMessageId` method might be better. Note that the `message_id` is a 64-bit Integer (see [Telegram Bot API > Message](https://core.telegram.org/bots/api#message)).

Alternatively , you can use the `getDate` method, which returns an Integer or DateTimeInterface (I don't know for sure), and check for a quote at or after that date.

Reply:
This does disable the feature to edit quotes. Maybe we could incorporate it in a later phase, if any demand for that is deemed usable.

Reply:
oef, should definitely be reworked then.

Reply:
Editing quotes via an edited message is currently not supported, but it does trigger the bot again. It results in duplicates, so it's okay to "break" this behaviour (it's actually "bug riding" so no issue if it no longer works)

Reply:
made it so that it looks at the message ID instead